src/main/java/core/basesyntax/RandomNumber.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomNumber {
```

Comment:
we don't need this class, this logic belongs to Lottery,  move random and this constant there